1-exercises/C-object-methods/exercise-3.js

```diff
@@ -12,9 +12,9 @@ let highScores = {
   100: ["David"],
 };
 
-let highScoreKeys = /* todo */
-let highScoreValues = /* todo */
-
+let highScoreKeys = Object.keys(highScores);/* todo */
```

Comment:
This line is right, but the lines below need fixing I'm afraid. There's a method on Object that works just like keys, but for values